ui/src/features/stage/promotions.tsx

```diff
@@ -171,6 +173,26 @@ export const Promotions = () => {
           </Link>
         </Tooltip>
       )
+    },
+    {
+      title: 'Extra',
```

Comment:
@rpelczar @krancour do you have any better name for this column?

Reply:
@Marvin9 You can leave it empty.